web/src/settings_invites.ts

```diff
@@ -103,13 +121,20 @@ function populate_invites(invites_data: {invites: Invite[]}): void {
                 item.expiry_date_absolute_time = timerender.absolute_time(item.expiry_date * 1000);
             }
             item.is_admin = current_user.is_admin;
-            item.disable_buttons =
+            item.disable_revoke_button =
                 item.invited_as === settings_config.user_role_values.owner.code &&
                 !current_user.is_owner;
+            item.disable_edit_button =
+                item.invited_as === settings_config.user_role_values.owner.code &&
+                !current_user.is_admin;
             item.referrer_name = people.get_by_user_id(item.invited_by_user_id).full_name;
             item.img_src = people.small_avatar_url_for_person(
                 people.get_by_user_id(item.invited_by_user_id),
             );
+            if (!settings_data.user_can_create_multiuse_invite()) {
+                item.disable_edit_button = true;
+                item.disable_revoke_button = true;
```

Comment:
Why revoke button needs to be disabled if user does not have creation permission?